scene/gui/button.cpp

```diff
@@ -286,6 +286,15 @@ void Button::_notification(int p_what) {
 					icon_size = Size2(icon_width, icon_height);
 				}
 
+				if (icon_set_height != 0) {
```

Comment:
Probably should be 
```suggestion
				if (icon_set_height > 0) {
```
to avoid negative values.

Reply:
Is the preferred workflow to use the "Commit suggestion" button or do a force push with all the changes?

Reply:
Amend + force push is preferred. (If there are a lot of suggestions you can also commit them all as one batch, then rebase to squash the commits.)